Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace test1
+{
+    class Program
+    {
+        //public List<Venta> ArraynuevaVenta = new List<Venta>();
+        //public List<Cliente> ArrayclienteNuevo = new List<Cliente>();
+        static void Main(string[] args)
+        {
+            
+             int _OpcionMenu = 0;            
+            while(_OpcionMenu!=6){
+                Console.WriteLine("--------------------");
+                Console.WriteLine("Menu de Facturacion");
+                Console.WriteLine("--------------------");
+                Console.WriteLine("1. Registro de Ventas");
+                Console.WriteLine("2. Registro de Clientes");
+                Console.WriteLine("3. Registro de vendedores");
+                Console.WriteLine("4. Registro de Productos");
+                Console.WriteLine("5. Reportes");
+                Console.WriteLine("6. Reportes");
+                Console.WriteLine("Seleccione una opcion");
+                _OpcionMenu = int.Parse(Console.ReadLine());
+                        
+                switch(_OpcionMenu){                     
+                    case 1: registroVentas();break;
+                    case 2: registroClientes();break;                
+                    case 3:registroVendedores();break;
+                    case 4:registroProductos();break;
+                    case 5:menuReporte();break;
+                    default : Console.WriteLine("Gracias por su visita");break;
+                }
+            }
+        }
+
+        public static void menuReporte(){
+            Console.WriteLine("--------------------");
+            Console.WriteLine("Menu de Reportes");
+            Console.WriteLine("1. Total de Ventas");
+            Console.WriteLine("2. Clientes mayores a 55 años de edad");
+            Console.WriteLine("3. Clientes que cumplen años en el mes");
+            Console.WriteLine("4. Productos con stock bajo.");
+        }
+
+        public static void reporteVentas(){
+
+            Console.WriteLine("Total de Ventas del dia :");
+
+        }
+
+        public static void reporteClientes(){
+
+            Console.WriteLine("Clientes mayores a 55 años:");
+            //foreach(Cliente client in ArrayclienteNuevo)

```

Comment:
again, you don't have to send your changes with code commented.

Reply:
ready